docs/releases.md

```diff
@@ -18,29 +18,46 @@ Welcome to **April 2021 release** :octicons-heart-16: of **DietPi**. It's an inc
 
     - [Merged software list](#select-software-from-a-merged-list)
 
-        The software selection is now simplified ! You have a single list to choose from, with over 190 software options !
+        The software selection is now simplified! You have a single list to choose from, with over 190 software options!
 
     - [New software](#new-optimised-software-in-71)
 
-        IPFS Nodes, Common UNIX Printing System (CUPS), and more in programming (Go language, and an excellent IDE: VSCodium)
+        IPFS Nodes, Common UNIX Printing System (CUPS), and more in programming (Go language, and an excellent IDE: VSCodium).
 
     - [DietPi tools: Additions & Major updates](#dietpi-tools-in-71-new-notable-updates)
 
-        DietPi supports more VPN clients and it has more Dynamic DNS options. Enabling SSL/TLS certificates is easier now using the updates brought to `DietPi-LetsEncrypt` tool. You can find more enhancements brought to: `DietPi-Drive_Manager`, `DietPi-Config`. 
+        DietPi supports more VPN clients and it has more Dynamic DNS options. Enabling SSL/TLS certificates is easier now using the updates brought to `DietPi-LetsEncrypt` tool. You can find more enhancements brought to: `DietPi-Drive_Manager`, `DietPi-Config`.
 
     - [Improvements](#changes-improvements-optimisations-in-71)
 
         More improvments to different software titles.
 
     - [Bug fixes](#bug-fixes) & updates to [supported SBC](#supported-sbc-updates-in-71) list
 
-Many thanks to all contributors and beta testers :octicons-heart-16: :octicons-heart-16: !! With your help we could achieve all of these updates ! Install or upgrade to this latest update - it has more things we hope you would enjoy ! Share your feedback on [Twitter](https://twitter.com/DietPi_), [DietPi Forum](https://dietpi.com/phpbb/), and report any issue to [GitHub](https://github.com/MichaIng/DietPi/issues)!
+Many thanks to all contributors and beta testers :octicons-heart-16: :octicons-heart-16:.  
+With your help we could achieve all of these updates! Install or upgrade to this latest update - it has more things we hope you would enjoy!  
+Share your feedback on [Twitter](https://twitter.com/DietPi_), [DietPi Forum](https://dietpi.com/phpbb/), and report any issue to [GitHub](https://github.com/MichaIng/DietPi/issues)!
+
+***Version #1:***  
+We already scheduled the next update:  
+***DietPi 7.2 will be launched on 2021-05-29***.
+
+***Version #2:***  
+
+!!! check "Next DietPi release"
+
+    As a new thing, we already scheduled the next update: ***DietPi 7.2 will be launched on 2021-05-29***.
```

Comment:
Just have something like this in mind: "As a new feature, we will introduce shorter release cycle and a fixed schedule. The next update is already planned: ***DietPi 7.2 will be launched on 2021-05-29***. "

Reply:
Agreed, that reads great. Only: `shorter release cycles` (plural)

Reply:
@Joulinar, @MichaIng: Added your remarks. 

What about the three versions?

Reply:
Personally I'm in favour of option 2. Probably we could introduce an image/logo later on. So we could have some more time looking into a layout and design?

Reply:
I also agree with option 2. Regarding the logo / image let's add them together in 7.2

Reply:
Ok, we start with version 2.